### PR TITLE
CSP-1345 add employer first and last name

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/SearchByPayeController.cs [hunk 1]
```diff
@@ -98,6 +98,15 @@ public async Task<IActionResult> Index([FromRoute] int ukprn, SearchByPayeSubmit
             return RedirectToRoute(RouteNames.AddEmployerMultipleAccounts, new { ukprn });
         }
 
+        var isInEAS = relationshipsRequest?.Account != null;
```
Comment (DevenKShah) [inline src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/SearchByPayeController.cs:101]: rename the var to read its purpose `hasExistingAccount`

File: src/SFA.DAS.Provider.PR.Web.UnitTests/Controllers/ContactDetails/ContactDetailsControllerPostTests.cs [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    private static readonly string Email = "test@account.com";
+
+    [Test, MoqAutoData]
+    public void Post_ReturnsExpectedViewModelAndPath(
+        Mock<IValidator<ContactDetailsSubmitViewModel>> validatorMock,
+     Mock<ISessionService> sessionServiceMock,
+     int ukprn,
+     string firstName,
+     string lastName,
+     CancellationToken cancellationToken
```
Comment (DevenKShah) [inline src/SFA.DAS.Provider.PR.Web.UnitTests/Controllers/ContactDetails/ContactDetailsControllerPostTests.cs:29]: alignment is off

**Review by DevenKShah — DISMISSED** [review]
Couple of minor suggestions

File: src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/SearchByPayeController.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/SearchByPayeController.cs:101]: ok

File: src/SFA.DAS.Provider.PR.Web.UnitTests/Controllers/ContactDetails/ContactDetailsControllerPostTests.cs (hunk 2 above)
Comment (MarkFCain) [reply src/SFA.DAS.Provider.PR.Web.UnitTests/Controllers/ContactDetails/ContactDetailsControllerPostTests.cs:29]: ok

**Review by DevenKShah — APPROVED** [review]

**Review by shaun-downey-education — APPROVED** [review]

**Pull request merged**